builder/functional_tests/clean-1.0/clean-1.0.dll

```diff
@@ -0,0 +1 @@
+Dummy file.
```

Comment:
The runtime builder checks that the entrypoint .dll exists to ensure that the app is in a good shape before generating the Dockerfile. It does not check that the file is in fact a .dll only that it exists.